deps/zlib/crc_folding.c

```diff
@@ -21,9 +21,11 @@
 #include <inttypes.h>
 #include <emmintrin.h>
 #include <immintrin.h>
+#include <smmintrin.h>
+#include <tmmintrin.h>
```

Comment:
I opened https://chromium-review.googlesource.com/c/chromium/src/+/2281008 for this

Reply:
The macro `CRC32_SIMD_SSE42_PCLMUL` is not set. It was disabled because the relevant code has been flagged as buggy. So I don't think that this change is needed.

Reply:
No longer necessary.